src/attribute.cc

```diff
@@ -0,0 +1,96 @@
+/* Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
You should change all the copyrights in the repo to say Copyright Istio Authors instead.